### DISCUSSION TITLE
please add option to AntiMicro to use outside of software and with-in windows OS desktop environment as a mouse and/or trackpad replacement.

### DISCUSSION THREAD

**Discussion by tadams587** [mayrinck/FOSSonMicrosoftStore#38] [Requests]
AntiMicro app is great for games. 

However, its missing option to use outside of software and with-in windows OS desktop environment. can you update it to use as a replacement mouse and/or trackpad on laptops? would be great if I could just pack my controller for my laptop & not my mouse as well.

Also, when using a button press as a replacement for scroll up and/or scroll down mouse wheel actions; seems a bit hyper sensitive. like in Minecraft java a single press scrolls through my active inventory a tad too quick. 

changing the scroll speed in mouse settings didn't slow it any...maybe because I don't have a mouse plugged in my laptop, so scroll speed must default to...sorry, I have no idea for this application. either way nothing I do corrects the hyper sensitivity of mouse wheel scrolling when mapping to a controller button press, not in windows settings or with-in the app settings. please fix?

**Comment by mayrinck** [comment]
Hello, @tadams587!

I'm not the developer of Antimicro, so I can't make any changes to how it works or its possibilities so far. If you need any new features I recommend you try AntimicroX, it's the successor to this software and has other new features, maybe the ones you've mentioned are there already.

**Reply by tadams587** [reply]
thanks for the info
